packages/cardano-services/test/Program/services/postgres.test.ts

```diff
@@ -127,12 +123,16 @@ describe('Service dependency abstractions', () => {
         config = { listen: { port } };
         apiUrlBase = `http://localhost:${port}/network-info`;
         epochMonitor = new DbSyncEpochPollService(db!, 1000);
+        lastBlockNoInDb = (await db!.query<BlockNoModel>(findLastBlockNo)).rows[0].block_no;
+        cardanoNode = mockCardanoNode(
+          healthCheckResponseMock({ blockNo: lastBlockNoInDb })
+        ) as unknown as OgmiosCardanoNode;
```

Comment:
This way of mocking will prepare our unit/integration tests when we no longer rely on test fixtures data, but work with local data generated by local-network e2e tests. It handles the introduced relationship between `db-sync` and `cardanoNode` related to the service's health check. (db sync should be synced with the node to determine healthiness)